v3/v3_service_test.go

```diff
@@ -6673,3 +6673,67 @@ func TestOperations_GroupsGetEntities(t *testing.T) {
 		})
 	}
 }
+
+func TestOperations_GetAvailabilityZone(t *testing.T) {
+	mux, c, server := setup(t)
+
+	defer server.Close()
+
+	mux.HandleFunc("/api/nutanix/v3/availability_zones/cfde831a-4e87-4a75-960f-89b0148aa2cc", func(w http.ResponseWriter, r *http.Request) {
+		testHTTPMethod(t, r, http.MethodGet)
+		fmt.Fprint(w, `{"metadata": {"kind":"availability_zone","uuid":"cfde831a-4e87-4a75-960f-89b0148aa2cc"}, "status":{"resources":{"management_url":"dfde831a-4e87-4a75-960f-89b0148aa2cc"}}}`)
```

Comment:
suggestion: we can also create a real response using keploy as done in karbon api tests. in karbon folder.

Reply:
Sure, we'll do it along with the next patch. Thank you!